op-chain-ops/genesis/config.go

```diff
@@ -240,7 +240,10 @@ type DeployConfig struct {
 	// UseFaultProofs is a flag that indicates if the system is using fault
 	// proofs instead of the older output oracle mechanism.
 	UseFaultProofs bool `json:"useFaultProofs"`
-
+	// UseCustomGasToken is a flag to indicate that a custom gas token should be used
+	UseCustomGasToken bool `json:"useCustomGasToken"`
```

Comment:
Parsing this needs to be added to the solidity deploy config. It should default to false if not set. Then the deploy script should check and use the configured address